app/Utils/feverUtil.php

```diff
@@ -9,8 +9,11 @@ class FreshRSS_fever_Util {
 	 * @return bool true if the path is writable, false otherwise.
 	 */
 	public static function checkFeverPath(): bool {
-		if (!file_exists(self::FEVER_PATH)) {
-			@mkdir(self::FEVER_PATH, 0770, true);
+		if (!file_exists(self::FEVER_PATH) &&
+			!mkdir($concurrentDirectory = self::FEVER_PATH, 0770, true) &&
+			!is_dir($concurrentDirectory)
+		) {
```

Comment:
The code before was better: the new one is not safe for concurrency, and also might generate warnings

Reply:
Ah, sorry, I read a bit too fast. The previous one was not so good either :-)